roles/ceph-common/files/cephstable.asc

```diff
@@ -0,0 +1,31 @@
+-----BEGIN PGP PUBLIC KEY BLOCK-----
```

Comment:
What is this used for? I'm not a huge fan of putting a key with valid format in a public repo. I'd rather it be malformed in some way so that it can't be used to accidentally (or on purpose) be put in some compromised host.